File: Problem1/Python/Problem1.py
# 1000보다 작은 자연수 중에서 3 또는 5의 배수를 모두 더하면?


def problem1():
    return sum(i for i in range(1000) if i % 3 == 0 or i % 5 == 0)

print(problem1())

File: Problem1/Java/byungtak/problem1.java
public class main {
	public static void main(String[] args) {
		int i = 0;
		int sum = 0;

		for(i=0; i<1000; i++) {
			if(i%5==0 || i%3==0) {
				sum += i;
			}
		}

		System.out.println(sum);
	}
}
